packages/mongodb/_dev/deploy/docker/docker-compose.yml

```
@@ -0,0 +1,27 @@
+version: '2.3'
+services:
+  mongodb:
+    build: .
+    ports:
+      - 27017
+    volumes:
+      - ./sample_logs:/var/log
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: example
+    entrypoint: ["mongod", "--bind_ip_all", "--replSet", "rs0"]
+    command: ['--logpath', '/var/log/mongodb.log']
+  mongodb-logfile:
+    image: alpine
+    volumes:
+      - ./sample_logs:/var/log
+      - ${SERVICE_LOGS_DIR}:/var/log/mongodb
+    command: /bin/sh -c "sleep 15 && chmod 777 /var/log/mongodb.log && cp /var/log/* /var/log/mongodb/"
```

Good idea to use a different container to initialize the log file :+1: 

Would it be another option to use this container to create the `/var/log/mongodb/mongodb.log`? This would allow to collect logs directly from the server logs instead of needing to copy the file to the shared volume.

It would be to:
* Modify the `mongodb` service to depend on `mongodb-logfile` (and remove the dependency in `mongodb-logfile`).
* Use `mongodb-logfile` to create the file with the proper permissions.
```suggestion
    command: /bin/sh -c "touch /var/log/mongodb/mongodb.log && chmod 777 /var/log/mongodb/mongodb.log"
```
* Start mongodb with `command: ['--logpath', '/var/log/mongodb/mongodb.log']`.

---

yeah right 
so i tried with different methods and concluded that mongodb.log should contains at least this much (-rw-r--r--) permissions.
so instead giving only logpath i also give log-append so it will use created mongodb.log file and now it's working.
Thanks